tests/tensorflow/keras/test_keras.py

```diff
@@ -341,7 +341,7 @@ def test_include_regex(out_dir):
     tr = create_trial_fast_refresh(out_dir)
     tnames = tr.tensor_names(collection="custom_coll")
 
-    assert len(tnames) == 12
+    assert len(tnames) == 8
```

Comment:
why this change?

Reply:
Because we stop passing gradients by default from TF to smdebug.

So there is no way to catch them by simply defining a regex